crate/operator/restore_backup.py

```diff
@@ -793,7 +793,8 @@ async def handle(  # type: ignore
 
             # Retrieve admin username from the CrateDB CRD
             cratedb = await get_cratedb_resource(namespace, name)
-            crd_username = cratedb["spec"]["users"][0]["name"]
```

Comment:
Why did this stop working? :thinking:

Reply:
It just made the test fail, because the test cluster used in the restore-backup test does not have any users defined when created (it did not need any for what it was testing till now). Should never happen on a real cluster.